howso/client/typing.py

```diff
@@ -37,6 +37,16 @@ class Evaluation(TypedDict):
     """A mapping of feature names to lists of values."""
 
 
+class Status(TypedDict):
```

Comment:
Lets call this TrainStatus instead. The keys also need to be set to t.NotRequired[bool] since they may not exist in the dict